test/unit/test-mysqlDaoQueryHelper.js

```diff
@@ -103,6 +104,36 @@ describe('MysqlDaoQueryHelper', function() {
             sql.should.eql("SELECT * FROM users WHERE (first_name IS NULL) AND (last_name = 'Doe') LIMIT 1");
         });
 
+        it('Should generate dates with for a utc timezone ', function() {
```

Comment:
I would add test cases in the "clean" section below in this file.  Also, I'd put the before/after stuff at the top of the describe block.